src/components/sykmelding/nysykmelding/NySykmelding.tsx

```diff
@@ -83,18 +91,19 @@ const NySykmelding: React.FC<SykmeldingProps> = ({ sykmelding }: SykmeldingProps
                 </Ekspanderbartpanel>
             </InfoPanel>
 
-            <div ref={utfyllingRef} className="third">
+            {/* TODO: Bestemme om denne skal være i Sporsmal-komponent eller som egen komponent */}
+            <div ref={utfyllingRef} style={{ marginTop: '2rem', marginBottom: '2rem' }}>
                 <h1>Bruk sykmeldingen</h1>
```

Comment:
Her mangler det en lenke og noen listelementer for at den skal bli lik gammel løsning.

Reply:
laget en komponent for det nå